hw/drivers/sensors/lis2dw12/src/lis2dw12.c

```diff
@@ -2549,25 +2567,76 @@ lis2dw12_sensor_set_config(struct sensor *sensor, void *cfg)
 static int
 lis2dw12_sensor_handle_interrupt(struct sensor *sensor)
 {
+    struct sensor_notify_os_ev *snoe;
     struct lis2dw12 *lis2dw12;
     struct sensor_itf *itf;
     uint8_t int_src;
+    uint8_t int_status;
     int rc;
 
+    (void)snoe;
```

Comment:
`snoe` never gets used.  Can it be removed from this function?

Reply:
yup, correct. sorry about that.